### ISSUE TITLE
frontend container startup script fails with BACKEND_BASE_URL set to an actual URL

### ISSUE THREAD

**Issue by syphar** [sartography/spiff-arena#1085]
We're trying to set up spiff arena and stumbled onto this issue. 

We're setting `SPIFFWORKFLOW_FRONTEND_RUNTIME_CONFIG_BACKEND_BASE_URL` to the backend base-URL, since it's not a subdomain of the frontend domain. 

Now we're seeing this error from the startup script: 
```
Backslash found where operator expected at -e line 1, near "app\"
syntax error at -e line 1, near "app\"
Can't find string terminator '"' anywhere before EOF at -e line 1.
Exited with BAD EXIT CODE '255' in /app/bin/boot_server_in_docker script at line: 38.
```

**Comment by syphar** [comment]
the value is something like `https://backend-xxx.a.run.app`

**Comment by burnettk** [comment]
are you running from a public docker image? which version? there was a relevant fix in commit `18789d9ea60bf2a7791612bb7563029188d3e39c`. please try upgrading if practical.

**Comment by burnettk** [comment]
also, if you have interest in using a saas-based version of the software that we host and maintain, please hit us up on the website, and we would treat you very well. :)

**Comment by syphar** [comment]
> are you running from a public docker image? which version? there was a relevant fix in commit `18789d9ea60bf2a7791612bb7563029188d3e39c`. please try upgrading if practical.

oh it's totally possible that the image is outdated, I'm trying to build a POC for self-hosting, and used the docker hub images (not by you: https://hub.docker.com/search?q=spiffworkflow ). 

How now, looking at it, 18789d9ea60bf2a7791612bb7563029188d3e39c is older than the images here. 

But in any case, I'll retest with the official images on github.

**Comment by syphar** [comment]
I now checked the github images, and the ones from above were quite outdated. 

I'm closing this issue, assuming that it's solved (I worked around it before creating this issue). If it's not solved, I reopen.
